packages/repository/test/unit/repositories/legacy-juggler-bridge.unit.ts

```diff
@@ -62,6 +62,33 @@ describe('DefaultCrudRepository', () => {
     }
   }
 
+  interface GeoPoint {
+    /**
+     * latitude
+     */
+    lat: number;
+
+    /**
+     * longitude
+     */
+    lng: number;
+  }
+  class Address extends Entity {
+    static definition = new ModelDefinition({
+      name: 'Address',
+      properties: {
+        mGeoPoint: {type: 'GeoPoint', required: true},
+        mBelongToId: 'number',
+        mId: {name: 'id', type: 'number', id: true, generated: true},
```

Comment:
I am curious, where is the `m` prefix coming from? I believe we don't use such prefixes elsewhere in LB4 codebase. Can you rename the properties please?

See also https://en.wikipedia.org/wiki/Hungarian_notation#Disadvantages

Reply:
It was coming from the model definition that our user @hosiduy had when he encountered the issue. I will rename them 👍.